pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -79,6 +79,16 @@ let this = buildPackages.symlinkJoin {
       done
     ''}
 
+    WRAPPER_NIX_CFLAGS_LINK_RUN=""
+    ${lib.optionalString stdenv.isDarwin ''
+      # Find all the Framework paths that must be available at link time
+      for x in "''${__pkgsTargetTarget[@]}" ; do
+        if [ -d "$x/Library/Frameworks" ] ; then
+          WRAPPER_NIX_CFLAGS_LINK_RUN+="-- -framework-path$x/Library/Frameworks"
```

Comment:
Does this work if there are other arguments coming later?  Usually `--` is a kind of once-only thing

Reply:
I'm not sure. It seems like both ways work, though they're processed in a different order?

```[nix-shell:~/repos/reflex-platform]$ runghc -- --info -- --show-options lol
ghc: on the commandline: cannot use `--info' with `--show-options'
on the commandline: cannot use `--info' with `-e'
on the commandline: cannot use `--info' with `-e'
Usage: For basic information, try the `--help' option.

[nix-shell:~/repos/reflex-platform]$ runghc -- --info --show-options lol
ghc: on the commandline: cannot use `--info' with `-e'
on the commandline: cannot use `--info' with `-e'
on the commandline: cannot use `--info' with `--show-options'
Usage: For basic information, try the `--help' option.

[nix-shell:~/repos/reflex-platform]$ runghc -- --ghc-arg=--info --ghc-arg=--show-options lol
ghc: on the commandline: cannot use `--info' with `-e'
on the commandline: cannot use `--info' with `-e'
on the commandline: cannot use `--info' with `--show-options'
Usage: For basic information, try the `--help' option.
```

Reply:
Any preference?
I think always using `--ghc-arg` would be the less ambigous thing

Reply:
Always using `--ghc-arg` sounds good to me.

Reply:
Done